src/main.js

```diff
@@ -27,6 +28,30 @@ Vue.use(VueSocketIO, socket);
 // Set up Vue Router
 Vue.use(VueRouter);
 
+// Set up PortalGun (connection to native app)
+PortalService.listen();
+
+window.addEventListener("load", () => {
+  // Let the app know the JS is ready
+  // FIXME: wait until vue is done with first render
+  PortalService.call("app.isLoaded");
```

Comment:
^^ ideally this would wait until after vue has finished first render so there's not a flash of white. idk if there's a hook into that?